clients/aws-sdk-transcribe-streaming/tests/integration/test_bidirectional_streaming.py

```diff
@@ -0,0 +1,112 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Test bidirectional streaming duplex event stream handling."""
```

Comment:
nit; we can probably shorten in this context. While duplex is subtly different than bidirectional, in this context they're representing the same concept. That we can communicate on the stream in both directions.

```suggestion
"""Test bidirectional event stream handling."""
```